supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from conftest import Fields, TestFields
+from supplier.models import Supplier
+
+
+class TestSupplierModel:
+    @pytest.mark.django_db()
+    def test_save_supplier(self, supplier0):
+        """Test - checks if supplier is saved in DB.
```

Comment:
Nice! but we don't need a full docstring in a test function. a good test can explain itself (and may need a comment here and there, if it is essential).

Reply:
i was taught to docstring every function/method, i see it as a good practice with good logic behind it, if it is really not essential i will take it down.

Reply:
usually not used on tests unless something REALLY needs explaining. and that should alarm us that the test isn't written well.

Reply:
ok, ill delete the docstring